### PR TITLE
Release notes for 8.0.13-4

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-8.0.13-4.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+|Percona Server| |release|
+================================================================================
+
+|Percona| announces the release of |Percona Server| |release| on
+|date| (downloads are available `here
+<https://www.percona.com/downloads/Percona-Server-8.0/>`__ and from
+the `Percona Software Repositories
+<https://www.percona.com/doc/percona-server/8.0/installation.html#installing-from-binaries>`__).
+This release contains a fix for a critical bug that prevented |Percona
+Server| 5.7.24-26 from being upgraded to version 8.0.13-3. |Percona
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.13-4.rst]: prevented from being upgrade if there were more than around 1000 tables, or if the maximum allocated InnoDB table ID was higher than around 1000. @satya-bodapati might have exact wording

File: doc/source/release-notes/Percona-Server-8.0.13-4.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+   `upgrade guide
+   <https://www.percona.com/doc/percona-server/8.0/upgrading_guide.html>`__ and the
+   document `Changed in Percona Server for MySQL 8.0
+   <https://www.percona.com/doc/percona-server/8.0/changed_in_version.html>`__.
+
+
+Bugs Fixed
+================================================================================
+
+- It was not possible to upgrade from MySQL 5.7.24-26 to 8.0.13-3. Bug fixed :psbug:`5245`.
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.13-4.rst]: Repeat same ID wording from above

File: doc/source/release-notes/Percona-Server-8.0.13-4.rst [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+- ``SHOW BINLOG EVENTS FROM <bad offset>`` is not diagnosed inside ``Format_description_log_events``. Bug fixed :psbug:`5126` (Upstream :mysqlbug:`93544`).
+- ``innodb.upgrade_orphan`` failed because of left files. Bug fixed :psbug:`5209` (Upstream :mysqlbug:`93686`).
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.13-4.rst]: Testsuite bug, goes to other bugs

File: doc/source/release-notes/Percona-Server-8.0.13-4.rst [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- There was a typo in `mysqld_safe.sh`: **trottling** was replaced with **throttling**. Bug fixed :psbug:`240`.
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.13-4.rst]: Thanks to Michael Coburn

File: doc/source/release-notes/Percona-Server-8.0.13-4.rst [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+- `main.mysqldump` failed because of dropped event. Bug fixed :psbug:`5268` (Upstream :mysqlbug:`93788`).
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.13-4.rst]: Testsuite bug, goes to other bugs

File: doc/source/release-notes/Percona-Server-8.0.13-4.rst [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+- ``EXPLAIN SELECT`` returned inconsistent number of ROWS in ``main.group_by``. Bug fixed :psbug:`5306` (Upstream :mysqlbug:`93703`).
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.13-4.rst]: Likewise

File: doc/source/release-notes/Percona-Server-8.0.13-4.rst [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+- On Debian or Ubuntu, a conflict was reported on the
+  :file:`/usr/bin/innochecksum` file when attempting to install |Percona Server| 8
+  over the |MySQL| 8. Bug fixed :psbug:`5225`.
+
+Other bugs fixed: :psbug:`4797`:, :psbug:`5264`:, :psbug:`5270`:, :psbug:`5309`:, :psbug:`5311`:
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.13-4.rst]: 5311: an out-of-bound read may occur on debug builds in the compressed columns with dictionaries feature

Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.13-4.rst]: 5308 (why it's not above?): a binlog compression-related memory leak in mysqlbinlog has been fixed

Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.13-4.rst]: 5307 (why not it's not above?): a compressed columns-related memory leak in mysqlbinlog has been fixed

Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.13-4.rst]: 5264: needs description

Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.13-4.rst]: 5222: a debug build might have crashed on upgrade from 5.7 with innodb_file_per_table=OFF

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-8.0.13-4.rst (hunk 1 above)
Comment (satya-bodapati) [reply doc/source/release-notes/Percona-Server-8.0.13-4.rst]: 1022 is exact InnoDB table_id. around 1000 is fine. So above wording by @laurynas-biveinis is OK

File: doc/source/release-notes/Percona-Server-8.0.13-4.rst [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+   <https://www.percona.com/doc/percona-server/8.0/upgrading_guide.html>`__ and the
+   document `Changed in Percona Server for MySQL 8.0
+   <https://www.percona.com/doc/percona-server/8.0/changed_in_version.html>`__.
+
+
+Bugs Fixed
+================================================================================
+
+- It was not possible to upgrade from MySQL 5.7.24-26 to 8.0.13-3 if
+  there were more than around 1000 tables, or if the maximum allocated
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.13-4.rst:40]: Either both "around 1000", either both 1022

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]
Only one comment out of many addressed

File: doc/source/release-notes/Percona-Server-8.0.13-4.rst (hunk 7 above)
Comment (borys-belinsky) [reply doc/source/release-notes/Percona-Server-8.0.13-4.rst]: Request to be added to Other bugs by Tyler Duzan

Comment (borys-belinsky) [reply doc/source/release-notes/Percona-Server-8.0.13-4.rst]: Request to be added to Other bugs by Tyler Duzan

File: doc/source/release-notes/Percona-Server-8.0.13-4.rst [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+  InnoDB table ID was greater than 1022. Bug fixed
+  :psbug:`5245`.
+- ``SHOW BINLOG EVENTS FROM <bad offset>`` is not diagnosed inside
+  ``Format_description_log_events``. Bug fixed :psbug:`5126` (Upstream
+  :mysqlbug:`93544`).
+- There was a typo in `mysqld_safe.sh`: **trottling** was replaced with
+  **throttling**. Bug fixed :psbug:`240`. Thanks to Michael Coburn for reporting
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.13-4.rst]: for the patch

File: doc/source/release-notes/Percona-Server-8.0.13-4.rst [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+  this error.
+- ``EXPLAIN SELECT`` returned inconsistent number of ROWS in
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.13-4.rst]: Other bugs, and add JIRA bug ID

File: doc/source/release-notes/Percona-Server-8.0.13-4.rst [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+- |Percona Server| 8.0 could crash with the "Assertion failure:
+  dict0dict.cc:7451:space_id != SPACE_UNKNOWN" exception during an
+  upgrade from |Percona Server| 5.7.23 to |Percona Server| 8.0.13-3
+  with ``--innodb_file_per_table=OFF``. Bug fixed :psbug:`5222`.
+- On Debian or Ubuntu, a conflict was reported on the
+  :file:`/usr/bin/innochecksum` file when attempting to install |Percona Server| 8
+  over the |MySQL| 8. Bug fixed :psbug:`5225`.
+- An out-of-bound read exception could occur on debug builds in the compressed
+  columns with dictionaries feature. Bug fixed :psbug:`5311`:.
+- The ``innodb_data_pending_reads`` server status variable contained an
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.13-4.rst:57]: Thanks to Fangxin Lou for the patch

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-8.0.13-4.rst [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+  incorrect value. Bug fixed :psbug:`5264`:. Thanks to Fangxin Lou for the patch.
+- A memory leak and needless allocation in compression dictionaries
+   could happen in ``mysqldump``. Bug fixed :psbug:`5307`.
+- A compression-related memory leak could happen in mysql.binlog. Bug
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.13-4.rst]: s/mysql.binlog/mysqlbinlog

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**